src/components/guide-tooltip/GuideTooltip.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../styles/variables';
+
+.bdl-GuideTooltip {
+    display: flex;
+}
+
+.bdl-GuideTooltip.bdl-Tooltip {
+    max-width: 408px;
+    padding: 24px 48px 24px 24px;
```

Comment:
can we update these padding/margins to use `bdl-grid-unit`?

Reply:
updated